include/hip/spirv_hip_fp16.h

```diff
@@ -1533,7 +1531,6 @@ THE SOFTWARE.
                     __llvm_amdgcn_rcp_f16(static_cast<__half_raw>(x).data)};
             }
 #endif
-            inline
```

Comment:
Accidental remove?

The `inline` keyword shouldn't be removed. As far as I can see, this would cause multiple definition errors.

Reply:
I see. To me the `if !defined(__HIP_PLATFORM_SPIRV__)` in the above is misplaced and it should encase the `inline` keyword that is just before it.